src/main/java/com/revlearn/team1/model/CourseModule.java

```diff
@@ -19,17 +20,22 @@ public class CourseModule {
 
     private String title;
 
+    private Long orderIndex;
```

Comment:
is it going to be a pain for each module to keep track of their own index in the course? I'll approve the PR either way but just wondering what you think on letting the Course handle their own ordering for simplicity sake?

Reply:
I do not know how the Course objects will order their arrays of modules.  My thinking was that the course modules are an outline of content for the course semester, and that instructors/educators would want them in a particular order.
Ideally we would implement a separate method to reorder modules so that none possess duplicate indices.